src/test/kotlin/wordle/StageTest.kt

```diff
@@ -0,0 +1,62 @@
+package wordle
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import wordle.domain.Stage
+import wordle.domain.Word
+
+internal class StageTest {
+
+    @Test
+    fun `스테이지를 만들면 PROGRESS`() {
+        val stage = Stage("hello")
+
+        assertThat(stage.state).isEqualTo(Stage.State.PROGRESS)
+        assertThat(stage.finished).isFalse()
```

Comment:
다중 Assertions 을 사용할 경우, 최초 실패한 Assertions 문제만 콘솔에 표현될 수 있습니다.
AssertAll 을 사용해보는 것은 어떠실까요? 모든 Assertions 이 실행되고 실패 지점을 한번에 파악할 수 있는 장점이 있습니다 :D

- [관련 링크](https://www.baeldung.com/junit5-assertall-vs-multiple-assertions) 

```suggestion
assertAll(
        { assertThat(stage.state).isEqualTo(Stage.State.PROGRESS) },
        { assertThat(stage.finished).isFalse() }
)
```